docs/content/framework/systems/Transfers/MultiAppCopyTransfer.md

```diff
@@ -1,9 +1,17 @@
-<!-- MOOSE Object Documentation Stub: Remove this when content is added. -->
-#MultiAppCopyTransfer
+# MultiAppCopyTransfer
+
 !devel /Transfers/MultiAppCopyTransfer float=right width=auto margin=20px padding=20px background-color=#F8F8F8
 
 !description /Transfers/MultiAppCopyTransfer
 
+## Description
+The MultiAppCopyTransfer allows for copying variables (both [nonlinear](auto::/Variables/Overview) and [auxiliary](auto::/AuxVariables/Overview)) between [MultiApps](auto::/MultiApps/Overview). All types of
+variables, including higher order, elemental, and nodal are supported. The only limitiation is that the
+meshes in the master and sub application must be identical.
```

Comment:
# of nodes and # elements should be sufficient.  If something still slips through that's your own damn vault...